plugins/polio/js/config.tsx

```diff
@@ -19,11 +19,11 @@ import WatchLaterIcon from '@mui/icons-material/WatchLater';
 import LocalShippingIcon from '@mui/icons-material/LocalShipping';
 import StorageIcon from '@mui/icons-material/Storage';
 import NotificationsActiveIcon from '@mui/icons-material/NotificationsActive';
-import { Dashboard } from './src/domains/Campaigns/Dashboard';
+import { Dashboard } from './src/domains/Campaigns/Dashboard/Dashboard';
```

Comment:
not a fan ...

Reply:
And I don't like `index` files 😅. I can try and look for a better name

Reply:
renamed the folder